RoeiJeRot.Logic/Services/ReservationService.cs

```diff
@@ -51,12 +51,14 @@ public interface IReservationService
     public class ReservationService : IReservationService
     {
         private readonly IBoatService _boatService;
+        private readonly IMailService _mailService;
         private readonly RoeiJeRotDbContext _context;
 
-        public ReservationService(RoeiJeRotDbContext context, IBoatService boatService)
+        public ReservationService(RoeiJeRotDbContext context, IBoatService boatService, IMailService mailService)
```

Comment:
De AllocateBoatReservations geeft een lijstje terug van de boten die niet opnieuw gereserveerd konden worden. Het is beter om de call naar SendCancelMail ergens buiten de Reservation Service te zetten, zoals in de view misschien.

Reply:
Zie RoeiJeRot.View.Wpf/Views/UserControls/BoatOverviewWindow.xaml.cs

Reply:
Ja, een service is een data aanvoer. Het is niet bedoeld om dit soort logica uit te voeren.